spec/tests/urbanopt/geojson/zoning_spec.rb

```diff
@@ -65,14 +67,18 @@
     expect(first_floor_points[0].class).to eq(OpenStudio::Point3d)
   end
 
-  # it 'gets first floor prints' do
-  #   # NOTE: MAKE THIS MORE SPECIFIC.
-  #   path = File.join(File.dirname(__FILE__), '..', '..', '..', 'files', 'nrel_stm_footprints.geojson')
-  #   feature_id = 'Energy Systems Integration Facility'
-  #   feature = URBANopt::GeoJSON::GeoFile.new(path).get_feature_by_id(feature_id)
-  #   feature_2 = URBANopt::GeoJSON::Zoning.handle_surrounding_buildings(@runner, @origin_lat_lon, feature)
-  #   expect(feature_2[0][]).to eq(feature.feature_json)
-  # end
+  it 'gets other building spaces' do
```

Comment:
Should I move this to building_spec.rb?

Reply:
I think this is okay here... since it is testing the "zoning=true"...